scripts/vanilla.zs

```diff
@@ -166,13 +166,22 @@ recipes.addShaped("fire_resistance_long", <minecraft:potion>.withTag({Potion: "m
 recipes.addShaped("xp_bottle", <minecraft:experience_bottle>, [
     [<minecraft:potion>.withTag({Potion: "minecraft:water"}), <ore:dustSugar>, <ore:dustTinyGold>]
 ]);
+mixer.recipeBuilder()
```

Comment:
This needs a space after the previous recipe.

Change this recipe to use a chemical reactor, and remove the sugar input, as an efficiency bonus for unlocking machines.